src/styles/_font-face-imports.scss

```diff
@@ -6,35 +6,35 @@
 
 @font-face {
   font-family: $fontSelawik;
-  src: url('./assets/fonts/selawk.eot');
-  src: url('./assets/fonts/selawk.eot?#iefix') format('embedded-opentype'),
-       url('./assets/fonts/selawk.ttf') format('truetype');
+  src: url('/assets/fonts/selawk.eot');
+  src: url('/assets/fonts/selawk.eot?#iefix') format('embedded-opentype'),
+       url('/assets/fonts/selawk.ttf') format('truetype');
 }
 
 @font-face {
   font-family: $fontSelawikBold;
-  src: url('./assets/fonts/selawkb.eot');
-  src: url('./assets/fonts/selawkb.eot?#iefix') format('embedded-opentype'),
-       url('./assets/fonts/selawkb.ttf') format('truetype');
+  src: url('/assets/fonts/selawkb.eot');
+  src: url('/assets/fonts/selawkb.eot?#iefix') format('embedded-opentype'),
+       url('/assets/fonts/selawkb.ttf') format('truetype');
 }
 
 @font-face {
   font-family: $fontSelawikLight;
-  src: url('./assets/fonts/selawkl.eot');
-  src: url('./assets/fonts/selawkl.eot?#iefix') format('embedded-opentype'),
-       url('./assets/fonts/selawkl.ttf') format('truetype');
+  src: url('/assets/fonts/selawkl.eot');
```

Comment:
Is it safe to change this to a root-level reference?  In most places, a leading '/' or '\\' indicates path from root, versus what it was './' which is relative path reference.